orchagent/main.cpp

```diff
@@ -574,6 +576,39 @@ int main(int argc, char **argv)
     attr.value.u64 = gSwitchId;
     attrs.push_back(attr);
 
+
+    char *platform = getenv("platform");
+    if (platform && strstr(platform, BRCM_PLATFORM_SUBSTRING) &&
```

Comment:
We can just have the check for voq, fabric alone ? 
Can remove the platform Broadcom check, I see Prince too suggested the same earlier.

Reply:
Fixed. Removed platfrom check.